### ISSUE TITLE
openup or spread option

### ISSUE THREAD

**Issue by FrankMittelbach** [Josef-Friedrich/cloze#1]
Looking at this package I was wondering if it is worth to provide some kind of "spread"  or "openup" option that enlarges the space of a cloze by a particular factor.

Rationale: if you do cloze tests then any participant actually using the text to fill in words will typically use handwriting, but that (if legible) is likely to need more space than the same text in typeset fonts.

Now one could use a larger clozefont to achieve this but then the "solution" document with the text inserted looks less good than one where the word spaces in cloze texts are larger (I think).

Just a thought - frank

**Comment by Josef-Friedrich** [comment]
Indeed, a “spread” option would be a nice enhancement for this package. Can you help to implement it? Can you provide a pull request?

Any thoughts how to implement? My thoughts: Register the `pre_linebreak_filter` callback and add after each glyph a kern node.

**Comment by FrankMittelbach** [comment]
> Indeed, a “spread” option would be a nice enhancement for this package.
> Can you help to implement it? Can you provide a pull request?

sorry, no time, too many other projects

> Any thoughts how to implement? My thoughts: Register the
> |pre_linebreak_filter| callback and add after each glyph a kern node.

that would give letter spacing (with its own ugliness) but yes that 
could be an option.

I was originally thinking more of measuring the widths of the cloze, 
calculate the spread width and then simply put half of that to the left 
and right of the cloze if the text is typeset (or distributed across all 
word spaces)
